staNMF/staNMF.py

```diff
@@ -371,10 +391,16 @@ def instability(self, k1=0, k2=0):
                 self.instabilitydict[k] = (np.sum(distMat) / (numReplicates *
                                            (numReplicates-1)))
 
-                outputfile = open("instability.csv", "w")
-                for i in sorted(self.instabilitydict):
-                    outputfile.write(str(i) + "," +
-                                     str(self.instabilitydict[i]) + "\n")
+                if self.parallel:
+                    outputfile = open(str(path + "instability.csv"), "w")
+                    outputfile.write("\n" + str(k) + "," + str(
+                                        self.instabilitydict[k]))
+                    output.close()
+        if not self.parallel:
+            outputfile = open("instability.csv", "w")
+            outputwriter = csv.writer(outputfile)
+            for i in sorted(self.instabilitydict):
+                outputwriter.writerow([str(i)], [str(self.instabilitydict[i])])
```

Comment:
sure, should work. Are you worried it'll mess something up by coercing it into a string?